Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -21,6 +21,17 @@ Write-Verbose -Verbose "configFile = $configFile"
 Write-Verbose -Verbose "dbConnectionString = $dbUrl"
 Write-Verbose -Verbose "dbUsername = $dbUsername"
 
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
+if ($prcaEnabled -ieq "true")
+{
+    Write-Verbose -Verbose "GitPullRequestCodeAnalysisEnabled is true, setting cmdline args for incremental mode for sonar-runner..."
+
+    if (!$cmdLineArgs.ToString().Contains("sonar.analysis.mode"))
+    {
+        Write-Verbose -Verbose "cmdlineargs doesn't contain sonar.analysis.mode"
```

Comment:
I'd make the casing of "cmdlineargs" the same as the variable name. The messages will also be end user visible in some scenarios, so I'd follow the normal rules user-visible output e.g. no contractions ("doesn't")